engine/include/spear/rendering/api.hh

```diff
@@ -8,7 +8,7 @@ enum API
 {
     OpenGL,
     Vulkan,
-    DirectX12,
+    DirectX, // DirectX 12 to be precise.
```

Comment:
What happens if some day DX13 implementation becomes a thing? This will be reverted then. I'm not going to do either of those so I don't care.